Source/WebCore/dom/make_names.pl

```diff
@@ -268,12 +274,18 @@ sub defaultAttrPropertyHash
     my $localName = shift;
 
     my $identifier = $localName =~ s/^x-webkit-/webkit/r =~ s/-/_/gr;
+    my $tagEnumValue = $identifier . "Attr";
```

Comment:
Since these are `enum class` values, we do not need the `Attr` suffix. I think `AttributeName::src` is fine and better than `AttributeName::srcAttr`. In `HTMLNames` we used the suffix to separate tag names from attribute names and so that we could do `using namespace HTMLNames` without many conflicts.

Reply:
This is actually problematic. In the enumeration, there are quite a few C++ keywords like `class` or `default`. The "Attr" suffix was a way for me to avoid this and I thought it looks OK since it was consistent with HTMLNames.

Otherwise, our perl script deals with C++ keywords by appending a `_` to the name but I thought AttributeName::class_ looked ugly.